third-party/rspamd/src/test/java/org/apache/james/rspamd/DockerClamAV.java

```diff
@@ -41,7 +42,8 @@ public DockerClamAV(Network network) {
             .withNetwork(network)
             .withCreateContainerCmdModifier(createContainerCmd -> createContainerCmd.withName("james-clamav-test-" + UUID.randomUUID()))
             .withNetworkAliases("clamav")
-            .waitingFor(Wait.forHealthcheck());
+            .waitingFor(Wait.forHealthcheck()
+                .withStartupTimeout(Duration.ofMinutes(2)));
```

Comment:
what do you think if we have a file configured somewhere, or read it from env?

Reply:
of course, it is not for only ClamAV

Reply:
I think a timeout env variable may help for different (slow/fast) environments.
But as long as we hardcode that timeout high enough, it is still fine for whatsoever environment. (fast env still finishes starting container regardless of the timeout)
Let's keep that in mind and let that out of this PR scope IMO.